### ISSUE TITLE
Jumping multiple page views, the pageviewcontroller goes white

### ISSUE THREAD

**Issue by adougies** [kitasuke/PagingMenuController#196]
Hey thanks for this repo. 
I am experiencing an issue however. It works great when selecting a different page which is next in the array, eg. 1 to 2. However when I skip a page e.g. 1 to 3. The view goes white for the animation and then shows the correct page.

Is there anything I can do to fix this or will it be resolved in a future update?

Thanks

**Comment by kitasuke** [comment]
Well, its the way it is. PagingMenuController loads only three view controllers(visible and neighbour ones), not all the view controllers. Do you want to load all the view controllers for the first time? Or do you simply want to fix white view with something else?

**Comment by adougies** [comment]
You said it loads the neighbour ones, but if i was going view 1 to 3, then 2 should already have been loaded? But yet it shows white on the animation. 
I would be happy with just it sliding in the selected view with the animation like it does to the neighbour controller, e.g. view 1 to 2.

**Comment by kitasuke** [comment]
Okay, it seems like a bug.
Could you tell me which version of PagingMenuController are you using? Also it would be helpful if you could show me your options for PagingMenuController.

**Comment by adougies** [comment]
I'm using 0.9.0. Here are my options:

```
let options = PagingMenuOptions()
        options.menuHeight = 44
        options.menuDisplayMode = .SegmentedControl
        options.backgroundColor = returnUIColor(festivalInfo.themeColour)
        options.selectedBackgroundColor = .clearColor()
        
        options.font = UIFont.systemFontOfSize(16)
        options.selectedFont = UIFont.boldSystemFontOfSize(16)
```

**Comment by kitasuke** [comment]
I've tried with the options you gave me, but example project works fine. Could you set the options to example project and see how it works?

**Comment by meanevo** [comment]
have the same issue while switching pages

**Comment by kitasuke** [comment]
It's fixed and merged in master branch. I'll let you know once it's released.

**Comment by kitasuke** [comment]
New version has just released! Please check this issue with 0.10.1 version.
